### DISCUSSION TITLE
Exclude code from run in design mode?

### DISCUSSION THREAD

**Discussion by sn4k3** [AvaloniaUI/Avalonia#7702] [Q&A] [answered]
Is there any variable i can use to prevent code from run in design mode? In Visual Studio AXMAL previewer?
Some code will break the preview if object is null or other complex data which depends on runtime

**Comment by jp2masa** [comment, marked as answer]
You can use `Design.IsDesignMode`.

**Reply by sn4k3** [reply]
Thanks, that's exactly what i want!
